src/resources/page-names.yml

```diff
@@ -13,6 +13,9 @@ settings:
 import:
   en: Import
   de: Import
+imports:
+  en: Imports overview
+  de: Importsübersicht
```

Comment:
Andreas' suggestion was for the other option https://github.com/educandu/educandu/pull/55#discussion_r759986889